tests/test_integrationToDoApp.py

```diff
@@ -27,73 +35,36 @@ def test_index_page(mock_get_requests, client):
     assert "testDoingTask_UID:1548993486125" in responseHtml
 
 
+def addMockData():
 
+    mongoDB=os.getenv('MONGO_DB_NAME')
+    mongoTable=os.getenv('MONGO_TABLE_NAME')
```

Comment:
I think you mean MONGO_COLLECTION_NAME instead of MONGO_TABLE_NAME

Your .env.template has both, but currently only TABLE_NAME is used and "COLLECTION" is the correct term.